pyaedt/modules/SolveSetup.py

```diff
@@ -2542,6 +2542,66 @@ def get_sweep(self, sweepname=None):
                 return self.sweeps[0]
         return False
 
+    @pyaedt_function_handler()
+    def get_sweeps(self):
```

Comment:
```suggestion
    def get_sweep_names(self):
```
I would change the name to have naming consistency with other PyAEDT methods